### PR TITLE
[4.1.x] Updated translations from Transifex.

### PR DESCRIPTION
**Pull request by felixxm**
This also fixes related i18n tests.

### CODE REVIEW

File: django/conf/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -13,15 +13,16 @@ msgid ""
...
+"POT-Creation-Date: 2022-05-17 05:23-0500\n"
+"PO-Revision-Date: 2013-04-25 06:49+0000\n"
+"Last-Translator: Claude Paroz <claude@2xlibre.net>, 2013-2022\n"
 "Language-Team: French (http://www.transifex.com/django/django/language/fr/)\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Language: fr\n"
-"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Plural-Forms: nplurals=3; plural=(n == 0 || n == 1) ? 0 : n != 0 && n % "
```
Comment (felixxm) [inline django/conf/locale/fr/LC_MESSAGES/django.po:24]: @claudep Do you want me to revert the `Plural-Forms` changes in `fr`?

Comment (claudep) [reply django/conf/locale/fr/LC_MESSAGES/django.po:24]: That was my intent, yes. I you want to try before I have time for it, feel free!

Comment (felixxm) [reply django/conf/locale/fr/LC_MESSAGES/django.po]: I pushed changes in the 2nd commit.

Comment (claudep) [reply django/conf/locale/fr/LC_MESSAGES/django.po]: Awesome, your are my champion !!

**Comment by claudep** [comment]
What desolates me so much is those changes:
```
- "PO-Revision-Date: 2017-09-19 21:51+0000\n"
- "Last-Translator: Carlos Muñoz <cmuozdiaz@outlook.com>\n"
+ "PO-Revision-Date: 2013-04-24 18:05+0000\n"
+ "Last-Translator: Carlos Muñoz <cmuozdiaz@outlook.com>, 2015\n"
```
which is due to bugs on Transifex side (po revision dates cannot go backwards, and the year should not be present in Last-Translator). I tried to discuss those with them some weeks ago, but it went nowhere. I'm fed up with Transifex now.

I might inspect this PR a bit more to see if it's feasible to fix that in some way.

**Comment by felixxm** [comment]
Closing in favor of #15905.

**Pull request closed without merging**